crates/sui-core/src/node_sync/node_follower.rs

```diff
@@ -112,6 +112,7 @@ impl EffectsStakeMap {
 
     pub fn forget_effects(&mut self, digests: &TransactionEffectsDigest) {
         self.effects_stake_map.remove(digests);
+        self.effects_vote_map.remove(digests);
```

Comment:
Not aware so much of SUI internal, but reading through the code makes sense to me. Is there a test that could be written to confirm the expectation of this?